css/style.css

```diff
@@ -0,0 +1,146 @@
+/* GENERAL STYLING */
+
+.crimson-font-color{
+  color: #540002;
+}
+
+.link, .link:visited{
+  color: #666F75;
+  opacity: 0.6;
+}
+
+.link:hover, .link:focus, .link:active {
+  text-decoration: none;
+  opacity: 1;
+  color: #636C72;
+}
+
+.item {
+  padding-bottom: 40px;
+}
+
+.page-head {
+  padding: 50px 0;
+}
+
+/* STYLE CONTAINERS */
+
+.cover .container{
+  padding-top: 200px; 
+}
+
+body, html {
+  height: 100%;
+}
+
+#about{
+  margin-bottom: 150px;
+  height: auto;
+  background: white;
+  color: #540002;
+}
+
+/* BUTTON PROPERTIES */
+
+.btn-info{
+  background: #74FF11;
+  color: #540002 !important;
+  border-color: #74FF11;
+}
+
+.btn-info:hover{
+  background: #64D612;
+  color: #540002 !important;
+  border-color: #64D612;
+}
+
+.btn{
+  padding: 10px 24px !important;
+  font-size: 18px;
+  margin: 3px 0 0 8px;
+  width: 130px;
+}
+
+/* NAVBAR COMPONENTS */
+
+.navbar{
+  background: transparent;
+}
+
+.navbar-toggler{
+  margin-top: 15px;
+  font-size: 25px;
+  border: none;
+}
+
+.navbar-brand{
+  font-size: 30px;
+}
+
+/* FOOTER ELEMENTS */
+
+.designer_name, .designer_name:visited{
+  text-decoration: none;
+  color: #767676;
+}
+
+.designer_name:hover, .designer_name:active, .designer_name:focus{
+  text-decoration: none;
+  opacity: 0.6;
+}
+
+/* #MAIN ELEMENTS */
+
+#main{
+  background: url(../imgs/main.jpg);
+  color: white;
+}
+
+#description {
+  opacity: 0.6;
+}
+
+.cover{
+  margin-right: 0;
+  margin-left: 0;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+}
+
+.row{
+  height: 100%;
+}
+
+#down, #down.hover, #down:active, #down:visited, #down:focus{
+  text-decoration: none;
+  color: #fff;
+  animation-delay: 3s;
```

Comment:
Make sure you know what browsers you are targeting. CSS animations are not supported by older browsers (like pretty much any version of IE). This is necessary for websites for many businesses because your users come from terrible browsers.

You should setup fallbacks for these features. Like the use of `modernizr` or browse-happy. Check out HTML5 Boilerplate to see how they do it.